collector/src/lib.rs

```diff
@@ -376,3 +378,23 @@ impl CollectorCtx {
 pub fn runtime_group_step_name(benchmark_name: &str) -> String {
     format!("runtime:{}", benchmark_name)
 }
+
+pub fn build_async_runtime() -> Runtime {
+    let mut builder = tokio::runtime::Builder::new_multi_thread();
+    // We want to minimize noise from the runtime
+    builder
+        .worker_threads(1)
+        .max_blocking_threads(1)
+        .enable_time()
+        .enable_io();
+    builder.build().expect("built runtime")
+}
+
+#[derive(Debug, clap::Args)]
+pub struct DbOption {
```

Comment:
Sorry, I probably wasn't clear. This is a CLI option, this should stay in `bin/collector`. What I meant was to create a function in `compare.rs` that would essentially look like this:
```rust
async fn compare_artifacts(conn: Box<dyn Connection>, base: String, modified: String)
```